packages/url_launcher/url_launcher_android/lib/url_launcher_android.dart

```diff
@@ -62,17 +62,42 @@ class UrlLauncherAndroid extends UrlLauncherPlatform {
     required Map<String, String> headers,
     String? webOnlyWindowName,
   }) async {
+    final WebViewOptions webViewOptions = WebViewOptions(
```

Comment:
There is a fair share of code duplication here, but it could only be resolved after https://github.com/flutter/flutter/issues/66721 unfortunately (could not make it a passthrough because there is no way to pass the useWebView parameter to launchUrl)